sycl/doc/extensions/supported/sycl_ext_oneapi_sub_group_mask.asciidoc

```diff
@@ -16,61 +17,71 @@
 // docbook uses c++ and html5 uses cpp.
 :language: {basebackend@docbook:c++:cpp}
 
-== Introduction
-IMPORTANT: This specification is a draft.
 
-NOTE: Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are trademarks of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc. used by permission by Khronos.
+== Notice
 
-This document describes an extension which adds a `sub_group_mask` type.  Such a mask can be used to efficiently represent subsets of work-items in a sub-group for which a given Boolean condition holds.
+[%hardbreaks]
+Copyright (C) 2021-2023 Intel Corporation.  All rights reserved.
 
-== Notice
+Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are trademarks
+of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc. used by
+permission by Khronos.
+
+
+== Contact
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/intel/llvm/issues
+
+
+== Dependencies
+
+This extension is written against the SYCL 2020 revision 6 specification.  All
+references below to the "core SYCL specification" or to section numbers in the
+SYCL specification refer to that revision.
 
-Copyright (c) 2021 Intel Corporation.  All rights reserved.
 
 == Status
 
-Working Draft
 
-This is a preview extension specification, intended to provide early access to a feature for review and community feedback. When the feature matures, this specification may be released as a formal extension.
+This is a proposed extension specification, intended to gather community
```

Comment:
@Pennycook, could you please take a look at this change specifically? This is a "supported" extension, but status only says that it is "proposed". I used this status to better match with the previous version, which has several comments that the spec is only a draft.

I'm not entirely sure which exact status is the right one here.

Reply:
Given that this is in the "supported" folder, I think the status is supposed to be:

> This extension is implemented and fully supported by DPC++.

@gmlueck - I had a quick look through the "supported" folder, and there are quite a few extensions that still have the old "Working Draft" wording (e.g. [sycl_ext_buffer_location](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/supported/sycl_ext_intel_buffer_location.asciidoc), [sycl_ext_intel_dataflow_pipes](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/supported/sycl_ext_intel_dataflow_pipes.asciidoc)).  Do we need somebody to go through all these extensions and update the status?

Reply:
Switched the status to "implemented" in 1aaaeca41d33dd1602bddaea3b17d82594d9b00c